README.md

```diff
@@ -1,15 +1,101 @@
 # getappmap/archive-appmap
 
+> To get started with AppMap in GitHub actions, you need to start by installing the [AppMap App on the official GitHub Marketplace](https://github.com/marketplace/get-appmap)
+
+> To see a step-by-step example of how to install this action into your software project, [review the official AppMap Documentation](http://appmap.io/docs/analysis/in-github-actions).
+
 GitHub action to build and store an AppMap archive.
 
-To see a step-by-step example of how to install this action into your software project, [review the official AppMap Documentation](http://appmap.io/docs/analysis/in-github-actions).
+## Table of contents
+
+- [Prerequisites](#prerequisites)
+- [Inputs](#inputs)
+- [Examples](#examples)
+- [Development](#development)
+
+## Prerequisites
+
+The `archive-action` needs to run AFTER your test cases have executed to build and save an AppMap archive to the GitHub build asset repository. Your application will need AppMap libraries and configuration files created in the project, or you will need to use the [`install-action`](https://github.com/getappmap/install-action) to setup AppMap before your test cases run. 
+
+For more information about how to setup AppMap in your project [review the official AppMap Documentation](http://appmap.io/docs/analysis/in-github-actions).
+
+## Inputs
+
+Add a step like this to your workflow:
+
+```yaml
+- name: Archive AppMaps
+  uses: getappmap/archive-action@v1
+  with:
+    # Command working directory. Change this if your project lives in a 
+    # subdirectory or for monorepo / multi-project support
+    # Default: '.'
+    directory: /path/to/code
```

Comment:
You can borrow this example from https://github.com/getappmap/install-action#inputs